mainwindow.cpp

```diff
@@ -41,7 +45,7 @@ void MainWindow::on_astroid_Button_clicked()
 ///function for when user clicks the spin value box to change values of the shape
 void MainWindow::on_spinScale_valueChanged(double scaler)
 {
-    this->ui->Shapearea->set_scale_size(scaler);
+
 }
 ///when the user clicks the astroid button
```

Comment:
Consistency:

when -> When

We can all brush up these small details up at the end or when we have time.